plugins/woocommerce/includes/class-wc-order.php

```diff
@@ -392,7 +392,23 @@ protected function status_transition() {
 
 		if ( $status_transition ) {
 			try {
-				do_action( 'woocommerce_order_status_' . $status_transition['to'], $this->get_id(), $this );
+				/**
+				 * Fires when order status is changed.
+				 *
+				 * @since x.x.x Added $status_transition parameter.
```

Comment:
I don't personally believe it's necessary to include a `@since` tag for the addition of the new parameters so feel free to drop that... but given we've touched this part of the codebase, our own checks won't let us _not_ document the hook with a `@since` tag. In that case, it'd be best to use the version this hook was originally introduced, which is `1.0.0`.